app/assets/stylesheets/welcome.scss

```diff
@@ -0,0 +1,51 @@
+$gold: #F8A803;
+
+.welcome-wrapper {
+  background-image: image-url('landing-page-pic.jpg');
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 100%;
+}
+
+.welcome-top-bar {
+  background: rgba(0,0,0,0);
+}
+
+.top-bar-section ul li a{
+  font-size: 4vmin;
+}
+
+.top-bar ul.left {
+  margin-left: 30px;
+}
+
+.welcome-search-form {
+  position: relative;
+  margin: 0 auto;
+  top: 25%;
+}
+
+.search-button {
+  display: none;
+}
+
+.tag-line {
+  color: $gold;
+  font-size: 7vmin;
+  text-align: center;
+  position: relative;
+  top: 75%;
+}
+
+#welcome-search-bar {
+  border-radius: 15px;
+  background-color: rgba(0,0,0,0);
+  border: 3px solid #fff;
+}
+
+#welcome-search-bar:focus {
+  border: 3px solid $gold;
+  background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.